src/app/gm-tools/feature/bet-bulk-cancel-page.component.ts

```diff
@@ -0,0 +1,193 @@
+import {
+  ChangeDetectionStrategy,
+  Component,
+  computed,
+  inject,
+  signal,
+} from '@angular/core';
+import { PageHeaderComponent } from '../../shared/ui/page-header.component';
+import { HeaderLinkComponent } from '../../shared/ui/header-link.component';
+import { PlayerService } from '../../shared/data-access/player.service';
+import { TableModule } from 'primeng/table';
+import { SkeletonModule } from 'primeng/skeleton';
+import { NgOptimizedImage } from '@angular/common';
+import { ButtonModule } from 'primeng/button';
+import { BetStatus, trackByPlayerId } from '../../shared/util/supabase-helpers';
+import { StronglyTypedTableRowDirective } from '../../shared/ui/strongly-typed-table-row.directive';
+import { BetService } from '../../shared/data-access/bet.service';
+import { PlayerModel, UserModel } from '../../shared/util/supabase-types';
+import { confirmBackendAction } from '../../shared/util/dialog-helpers';
+import { ConfirmationService, MessageService } from 'primeng/api';
+
+@Component({
+  selector: 'joshies-bet-bulk-cancel-page',
+  standalone: true,
+  imports: [
+    PageHeaderComponent,
+    HeaderLinkComponent,
+    TableModule,
+    SkeletonModule,
+    NgOptimizedImage,
+    ButtonModule,
+    StronglyTypedTableRowDirective,
+  ],
+  template: `
+    <joshies-page-header headerText="Bulk Cancel Bets" alwaysSmall>
+      <joshies-header-link
+        text="GM Tools"
+        routerLink=".."
+        chevronDirection="left"
+      />
+    </joshies-page-header>
+
+    <p class="mt-5">Cancel all bets that are pending or (optionally) active.</p>
```

Comment:
The instructions should clarify that these buttons only affect bets where the player is the `requester`, not bets where the player is the `opponent`.

Since this says "all bets," my initial assumption was that it includes both sets (bets where the player is the `requester` or the `opponent`).